packages/blocks/src/multiple-choice-answer/index.js

```diff
@@ -64,4 +64,6 @@ const MultipleChoiceAnswer = ( { attributes, className } ) => {
 	);
 };
 
+MultipleChoiceAnswer.BlockName = 'crowdsignal-forms/multiple-choice-answer';
```

Comment:
Can we make these snakeCase? Like, it's just a string, not a component.

Reply:
🐫 case

Reply:
Sorry, not clear to me, are we going with `blockName` or `BlockName`? If it's just a string I'll use `blockName`

Reply:
No, it should actually be █Name.

Reply:
Well, now I got confused 😅.
@ice9js are suggesting something like `MultipleChoiceAnswer.Name`?
I thought it was `MultipleChoiceAnswer.blockName`.
My first attempt was to use `MultipleChoiceAnswer.name` (only `name`, lowercase). 
But the object `MultipleChoiceAnswer` is a function that already has the `name` property, so I can't use it.

Reply:
We settled to export it as `blockName`